octopoes/tests/robot/CeleryMonitor.py

```diff
@@ -84,6 +92,7 @@ def stop_monitoring(self):
     def count_tasks(self, event_type: str) -> int:
         event_type = "task-" + event_type.lower()
 
+        logger.info(self._monitor.sum_by_type)
```

Comment:
Perhaps add some text to make clear what the output is? Also, the `msg` parameter is usually a string

Reply:
Implemented :) Robot does not allow standard logger formatting (`'%s'` notation), so I used a n f-string instead